homeassistant/components/notify/llamalab_automate.py

```diff
@@ -0,0 +1,62 @@
+"""
+LlamaLab Automate notification service.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/notify.llamalab_automate/
+"""
+import logging
+import requests
+import voluptuous as vol
+
+from homeassistant.components.notify import BaseNotificationService
+from homeassistant.const import (CONF_API_KEY, CONF_NAME, CONF_PLATFORM)
+
+_LOGGER = logging.getLogger(__name__)
+_RESOURCE = 'https://llamalab.com/automate/cloud/message'
+
+CONF_TO = 'to'
+CONF_DEVICE = 'device'
+
+PLATFORM_SCHEMA = vol.Schema({
+    vol.Required(CONF_PLATFORM): "llamalab_automate",
+    vol.Optional(CONF_NAME):  vol.Coerce(str),
+    vol.Required(CONF_API_KEY):  vol.Coerce(str),
+    vol.Required(CONF_TO):  vol.Coerce(str),
+    vol.Optional(CONF_DEVICE, default=""):  vol.Coerce(str),
```

Comment:
By marking it optional, the default will be `None` if not specified. That way you won't have to do line 34-35